suite/tests/client-interface/drx-scattergather.dll.c

```diff
@@ -179,6 +160,26 @@ event_bb_app2app(void *drcontext, void *tag, instrlist_t *bb, bool for_trace,
     }
     CHECK((scatter_gather_present IF_X64(&&expanded)) || (expansion_ok && !expanded),
           "drx_expand_scatter_gather() bad OUT values");
+    if (one_time_mask_clobber_test) {
+        for (instr = instrlist_first(bb); instr != NULL; instr = instr_get_next(instr)) {
+            if (instr_get_opcode(instr) == OP_kandnw) {
+                instr_t *search_app_instr = instr;
+                while (!instr_is_app(search_app_instr)) {
+                    search_app_instr = instr_get_next(search_app_instr);
```

Comment:
This loop can be replaced with instr_get_next_app(), right?

Reply:
Neat, didn't know the function exists. Thanks.

Reply:
Done.